Tools/TestWebKitAPI/Tests/WGSL/ConstLiteralTests.cpp

```diff
@@ -43,38 +42,36 @@
     NumberType expectedValue;
 };
 
-@interface ConstLiteralTests : XCTestCase
+namespace TestWebKitAPI {
 
-@end
-
-@implementation ConstLiteralTests
-
-- (void)setUp {
-    [super setUp];
-    [self setContinueAfterFailure:NO];
+TEST(WGSLConstLiteralTests, DeleteBeforeLanding)
```

Comment:
To test that TestWGSL is running on EWS, this should fail. Will remove before merging.